src/Promotions/ServiceProvider.php

```diff
@@ -14,6 +14,7 @@
 use Give\Promotions\InPluginUpsells\PaymentGateways;
 use Give\Promotions\InPluginUpsells\RecurringDonationsTab;
 use Give\Promotions\InPluginUpsells\SaleBanners;
+use Give\Promotions\InPluginUpsells\SummerSalesBanner;
```

Comment:
We also have a few imports here using paths with nothing there, here a the list:

```
use Give\Promotions\FreeAddonModal\Controllers\DisplaySettingsButton;
use Give\Promotions\FreeAddonModal\Controllers\EnqueueModal;
use Give\Promotions\FreeAddonModal\Controllers\PreventFreshInstallPromotion;
use Give\Promotions\InPluginUpsells\RecurringDonationsTab;
```

Reply:
Resolved: https://github.com/impress-org/givewp/commit/b5deda101c5cd14a8969b7529ddd4037f19e5c47